src/components/snackbar/snackbar.css

```diff
@@ -0,0 +1,32 @@
+.snackbar {
+	border-radius: 5px;
+	background-color: var(--COLOR-PRIMARY);
+	color: var(--COLOR-WHITE);
+	font-weight: 400;
+}
+
+.snackbar__text {
+	padding: 1.4rem 1.6rem;
+}
+
+.snackbar button {
+	line-height: 3.6rem;
+	color: inherit;
+	opacity: 0.8;
+}
+
+.snackbar button:hover {
+	box-shadow: 0px 0px 1px 1px var(--COLOR-BOX-SHADOW4),
+		0px 0px 1px 5px var(--COLOR-BOX-SHADOW5);
+	opacity: 1;
+}
+
+.snackbar .btn--contained {
+	width: 6.4rem;
+	padding: 0 0.8rem;
+}
+
+.snackbar .btn--close--transparent {
+	margin-left: 0.8rem;
```

Comment:
better to use px for values less than 10px and also for margin and padding

Reply:
Done!